doc/apiv3-documentation.apib

```diff
@@ -3492,10 +3803,10 @@ For more details and all possible responses see the general specification of [Fo
 
 ## Available watchers [GET]
 
-Gets a list of users that can watch the work package.
+Gets a list of users that are able to be watchers of the specified work package.
 
 + Parameters
-    + work_package_id (required, integer, `1`) ... Work package id
+    + id (required, integer, `1`) ... work package id
```

Comment:
As stated above:

Use the shortest possible name and only prepend words if that is neccessary.

If one wanted to change that, I would propose a separate PR that changes it consistently across the board.